release/cloudbuild.yaml

```diff
@@ -116,6 +116,7 @@ steps:
       - '{"origin": "${_ORIGIN}", "bucket": "${_LOG_NAME}", "kmsKeyName": "ft-log-prod", "kmsKeyRing": "firmware-release-prod", "kmsKeyVersion": 1, "kmsKeyLocation": "global", "noteKeyName": "transparency.dev-aw-ftlog-prod"}'
 substitutions:
   # Build-related.
+  _ARMORED_WITNESS_REPO_VERSION: e7141b6db638c3a2cb23e354cedd2d2980d0fb3a
```

Comment:
How/when will this pin be moved forward? Is pinning to `@main` an option?

The real solution here is that we properly release the commands in that repo and pin to release tags here, but I'm not sure we're ready for that yet.

Reply:
I noticed we added the pinning to cloudbuild_ci.yaml, so I just added them here to be consistent. I agree that we should pin to release tags. I think it's fine to leave the commit for now, especially since we aren't currently developing `cmd/manifest` or `cmd/sign` anymore. WDYT?

Reply:
BTW, I reverted this change so that we can make the decision in another PR. It sounds like we need to think about this more carefully rather than sneak it in as as add-on to this PR.